<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QSpinBox" name="xPercentageSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>800</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="xSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>800</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="yPercentageSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>800</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="xPixelSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>32000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="changeNothingRadioButton">
     <property name="text">
      <string>change nothing</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="percentageRadioButton">
     <property name="text">
      <string>Size in %</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QRadioButton" name="pixelRadioButton">
     <property name="text">
      <string>Size in Pixel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QSpinBox" name="yPixelSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>32000</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QComboBox" name="sizeRatioComboBox">
     <item>
      <property name="text">
       <string notr="true">1:1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">5:4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">4:3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">16:9</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="ySlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>800</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QComboBox" name="pixelComboBox">
     <item>
      <property name="text">
       <string notr="true">VGA (640x480)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">XGA (1024x768)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">NTSC (576x486)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">PAL (720x486)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">HD (1920x1080)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Flickr large (768x1024)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">iPhone landscape (480x320)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">160x160</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">HD 720 (1280x720)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SVGA (800x600)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">PSP (480x272)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">2K (2048x1080)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>xSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xPercentageSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xPercentageSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yPercentageSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yPercentageSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>percentageRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>xSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>percentageRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>percentageRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>xPercentageSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>percentageRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>yPercentageSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pixelRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>xPixelSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pixelRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>yPixelSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
